### PR TITLE
First part of Task Estimation in Scrum

### PR DESCRIPTION
**Pull request by JouharaAlwardi**
Task Estimation: Introduction, Building a successful story point estimation and The importance of story point estimation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,81 @@
...
+
+
+
+&nbsp;
+
+<img src="https://github.com/xtreger/handbook-of-best-practices/blob/task-estimation-in-scrum-jouhara/Images/steps.png" width="500" height="200"/>
+
+&nbsp;
+
+### 2. The importence of story point estimation
```
Comment (diyarenny) [inline README.md]: Can you correct the spelling of importance?

Comment (diyarenny) [inline README.md]: Same as above

Comment (JouharaAlwardi) [reply README.md]: Done

**Review by diyarenny — APPROVED** [review]
Very well structured and easy to read. I'm approving

File: README.md [hunk 2]
```diff
@@ -1,2 +1,81 @@
...
+| Prt    | quantity of productive hours (usually 6 hours) |
+
+
+
+&nbsp;
+
+|References |
+|---|
+|[Building a successful story point estimation](http://www.agilebuddha.com/agile/agile-estimation-8-steps-to-successful-story-point-estimation/)|
+[The importence of story point estimation](https://rubygarage.org/blog/3-reasons-to-estimate-with-story-points)|
```
Comment (xtreger) [inline README.md]: One more small mistake here that Diya mentioned earlier.

**Review by xtreger — CHANGES_REQUESTED** [review]
Looks good, just one more small mistake to be fixed.

File: README.md (hunk 2 above)
Comment (JouharaAlwardi) [reply README.md]: Fixed

**Review by xtreger — APPROVED** [review]
Looks good, approving PR.

**Pull request merged**